src/com/mellanox/cicd/Matrix.groovy

```diff
@@ -869,10 +869,12 @@ spec:
         volumes: listV
     )
     {
-        node(POD_LABEL) {
-            stage (branchName) {
-                container(cname) {
-                    runSteps(image, config, branchName, axis, steps, 'k8')
+        retry(count: 2, conditions: [kubernetesAgent(), nonresumable()]) {
```

Comment:
Which real failure case do you prevent with such retry?

Reply:
The motivation for it is IL Blossom k8s issue described here: https://nvbugswb.nvidia.com/NvBugs5/SWBug.aspx?bugid=5115436&cmtNo=

We hope that it will reduce the impact when our PODs are terminated unexpectedly during a pipeline